.azure-pipelines/azure-pipelines-linux.yml

```diff
@@ -12,32 +12,27 @@ jobs:
         CONFIG: linux_64_c_compiler_version10cuda_compiler_version11.2cxx_compiler_version10python3.10.____cpython
         UPLOAD_PACKAGES: 'True'
         DOCKER_IMAGE: quay.io/condaforge/linux-anvil-cuda:11.2
-        SHORT_CONFIG: linux_64_c_compiler_version10cuda_c_hce12f117c9
       linux_64_c_compiler_version10cuda_compiler_version11.2cxx_compiler_version10python3.11.____cpython:
         CONFIG: linux_64_c_compiler_version10cuda_compiler_version11.2cxx_compiler_version10python3.11.____cpython
         UPLOAD_PACKAGES: 'True'
         DOCKER_IMAGE: quay.io/condaforge/linux-anvil-cuda:11.2
-        SHORT_CONFIG: linux_64_c_compiler_version10cuda_c_he3386405e7
       linux_64_c_compiler_version10cuda_compiler_version11.2cxx_compiler_version10python3.9.____cpython:
         CONFIG: linux_64_c_compiler_version10cuda_compiler_version11.2cxx_compiler_version10python3.9.____cpython
         UPLOAD_PACKAGES: 'True'
         DOCKER_IMAGE: quay.io/condaforge/linux-anvil-cuda:11.2
-        SHORT_CONFIG: linux_64_c_compiler_version10cuda_c_hfab6938474
       linux_64_c_compiler_version12cuda_compiler_versionNonecxx_compiler_version12python3.10.____cpython:
         CONFIG: linux_64_c_compiler_version12cuda_compiler_versionNonecxx_compiler_version12python3.10.____cpython
         UPLOAD_PACKAGES: 'True'
         DOCKER_IMAGE: quay.io/condaforge/linux-anvil-cos7-x86_64
-        SHORT_CONFIG: linux_64_c_compiler_version12cuda_c_h7bcdd601dd
       linux_64_c_compiler_version12cuda_compiler_versionNonecxx_compiler_version12python3.11.____cpython:
         CONFIG: linux_64_c_compiler_version12cuda_compiler_versionNonecxx_compiler_version12python3.11.____cpython
         UPLOAD_PACKAGES: 'True'
         DOCKER_IMAGE: quay.io/condaforge/linux-anvil-cos7-x86_64
-        SHORT_CONFIG: linux_64_c_compiler_version12cuda_c_h5795ae2e9a
       linux_64_c_compiler_version12cuda_compiler_versionNonecxx_compiler_version12python3.9.____cpython:
         CONFIG: linux_64_c_compiler_version12cuda_compiler_versionNonecxx_compiler_version12python3.9.____cpython
         UPLOAD_PACKAGES: 'True'
         DOCKER_IMAGE: quay.io/condaforge/linux-anvil-cos7-x86_64
-        SHORT_CONFIG: linux_64_c_compiler_version12cuda_c_h653c10ac84
+  swapfile_size: 10GiB
```

Comment:
Not sure why, but `swapfile_size` is being added here

This results in Azure being unable to parse the YAML and so Azure doesn't run. Instead it shows [this error]( https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=772819&view=results ):

```
/.azure-pipelines/azure-pipelines-linux.ymI(Line:35, Col: 3): Unexpected value 'swapfile_size'772819
```

<br>

Unfortunately it does even report the parsing error back to GH (making it seem like things passed 😞)

Reply:
cc @mikemhenry

Reply:
Should be fixed by PR: https://github.com/conda-forge/conda-smithy/pull/1759